tests/e2e/script_test.go

```diff
@@ -49,6 +52,12 @@ func TestE2EScript(t *testing.T) {
 		cancel()
 	})
 
+	type gvkByNN struct {
```

Comment:
Nit: technically it's GVKNN, (no "by"), which is actually an acronym I've used before.  You could use GroupVersionKindNamespaceName if you were worried that GVKNN was not very clear, but personally I think it's very clear once you're told and GVKNN is a little shorter to type & read!